docs/src/dev_guide/inputs.md

```diff
@@ -0,0 +1,337 @@
+# Inputs
+
+Inputs used to run the holistic electricity model: [indices](#indices) and [parameters](#parameters). [Indices](#indices) refer to the dimensions using which the rest of the data is organized, such as the number of generation and storage resources, independent power producers, the balancing areas condsidered, the representative days considered etc.
+
+## Indices
+
+### Common Indices
+
+ - **index_y.csv**: Representative simulation years. It consisits of a single row of year values. The file is used to generate two types of indices, `index_y` which is the years for a simulation window and `index_y_fix` which represents the full simulation horizon (does not change).
+
+ - **index_s.csv**: New resource depreciation years represented as a row vector.
+
+ - **index_d.csv**: Representative day and hour (from ReEDS). Each day is represented as `d[0-9][0-9][0-9]` in a row vector.
+
+ - **index_t.csv**: Representative hour within each representative day (from ReEDS) represented as `h[0-9][0-9][0-9]` in a row vector.
+
+ - **index_h.csv**: Customer groups. Example values are `Residential, Commercial, Industrial`.
+
+ - **index_j.csv**: Green technology types. Example values are `biopower, wind-ons, wind-ofs`.
+
+ - **index_z.csv**: Balancing Area zones as modeled in ReEDS. Refer to the [model regions map](https://www.nrel.gov/analysis/assets/pdfs/reeds-model-regions-map.pdf).
+
+ - **index_sector.csv**: Customer high level groups. Example values are `Residential, Commercial, Industrial`.
+
+ - **index_h_sector_mapping.csv**: File that maps each zone in index\_z to a group in index\_sector. It conisits of two columns, `index_z` and `index_h`.
+
+### Agent-Specfic Indices
+
+#### Regulator
+
+ - **index_rate_tou.csv**: Index for supported time-of-use rates. Example values are `Off-Peak, On-Peak, Mid-Peak`.
+
+ - **tou_rate_structure_\$(tou_suffix).csv**: Time-of-use rate for each representative hour and representative day combination. The column names are `index_d`, `index_t` and `index_rate_tou`. Here, the input file selected depends on the `tou_suffix` specified in the config.yaml file under `regulator_options`.
+
+#### VIU/IPP
+
+ - **index_k_existing.csv**: Existing bulk generation technologies. Consists of a single row containing the existing technolgies, for e.g., `coaloldscr`, `gas-ct`, `nuclear`, `wind-ofs` etc.
```

Comment:
```suggestion
 - **index_k_existing.csv**: Existing bulk generation technologies, e.g., `coaloldscr`, `gas-ct`, `nuclear`, `wind-ofs` etc.
```

Reply:
(And delete other instances of `for` prior to `e.g.`, because e.g. stands for "for example".)